integration_tests/suite/test_event_handler.py

```diff
@@ -515,3 +515,30 @@ def channel_unheld():
                 )
             ),
         )
+
+    @fixtures.db.user()
```

Comment:
you should explicitly set dnd to False when it's what you test
```suggestion
    @fixtures.db.user(do_not_disturb=False)
```